src/qunit.css

```diff
@@ -68,6 +68,12 @@
 	overflow: hidden;
 }
 
+#qunit-filteredTest {
+	padding: 0.5em 1em 0.5em 1em;
+	background-color: #F4FF77;
+	color: #366097;
```

Comment:
Running filtered tests already makes the reporter look different, the badge color might be improved later as I believe the current is not that bad.